gh-action/action.yml

```diff
@@ -19,6 +19,9 @@ inputs:
   stage-poggit:
     description: "Stage pharynx output to a new branch for Poggit"
     default: "true"
+  additional-assets:
+    description: "Additional assets other than LICENSE, .poggit.yml and plugin.yml, separated by colon, works the same as 'git add <pathspec>'"
```

Comment:
why separate by colon? isn't it one file per line? or did you mean source and dest separated by colon?

also don't refer to git pathspec; it's much more complicated than this

Reply:
I mean it is not indicated in the description anywhere that it is file per line.

Never mind about the `git add` part; I forgot I exec'ed git directly there.

Reply:
I will change it to that then and I guess the description of `additional-sources` should be changed too but it is outside of the scope of this PR